en/micro-integrator/docs/references/connectors/connector-usage.md

```diff
@@ -0,0 +1,159 @@
+# Connector Usage Guidelines
+
+This document provides a set of guidelines on how to use connectors throughout their lifecycle.
+
+## Using connectors in your integration project
+
+Connectors can be added and used as part of the integration logic of your integration solution. This helps you configure inbound and outbound connections to third-party applications or to systems that support popular B2B protocols.
+
+### Importing connectors 
+
+All the connectors are hosted at [WSO2 EI Connector Store](https://store.wso2.com/store/assets/esbconnector/list). You can download the connector from the store as a .zip file. 
+
+<img src="../../../assets/img/connectors/connector-store.png" title="Connector store" width="700" alt="Connector store"/>
+
+The source code for connectors can also be found in the specific [WSO2 extensions GitHub repository](https://github.com/wso2-extensions/).
+
+However, the recommended approach to use connectors for integration logic development is through WSO2 Integration Studio. Developers can browse and import connectors to the workplace using the Integration Studio itself. As a result, there is no need to go and download the connector from the store separately or obtain it from the source code.
+
+**To import a connector**:
+
+1. Open [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/).
+
+2. [Create an Integration Project](../../develop/create-integration-project.md).
+
+3. Right-click on the ESB Configs folder and select **New** -> **Add/Remove Connector**. Search for the connector and follow the steps in the wizard to import the connector.
+
+    <img src="../../../assets/img/connectors/import-connector.png" title="Import a connector" width="700" alt="Import a connector"/>
+
+### Providing values for operation parameters 
+
+After importing the connector, you can drag and drop operations to the design palette and use them. When providing values for operation parameters, you can provide static values or dynamic values. Dynamic values can be provided in one of the following ways. 
+
+* As an [XPATH expression](https://www.w3schools.com/xml/xpath_syntax.asp)
+* As a [JSON expression](https://docs.oracle.com/cd/E60058_01/PDF/8.0.8.x/8.0.8.0.0/PMF_HTML/JsonPath_Expressions.htm) 
+* As a property. 
+    * Most of the time this will be a custom property you set earlier in the mediation flow. The property exists throughout the mediation flow. 
+    * You can also provide properties of other scopes as well (i.e., a header value).
+
```

Comment:
in the mediation flow using property mediator (link to property mediator). Any property set with default scope exists throughout the message flow and you can read it anywhere in the message flow after it is set. 

You can also provide properties of other scopes as well (i.e., a header value). However they may not exist throughout the message flow. Please read property mediator documentation to understand more.